imports/ui/explorer-components/item-list.jsx

```diff
@@ -23,25 +23,25 @@ const updateCounter = (iconType, count) => {
   )
 }
 
-export function CaseList ({ allCases, onItemClick }) {
+export function ItemList ({ allItems, onItemClick, name }) {
   return (
     <div>
-      {allCases.map((caseItem, ind) =>
-        <li key={caseItem.id} className='h2-5 bt b--black-10'>
+      {allItems.map((item, ind) =>
+        <li key={item.id} className='h2-5 bt b--black-10'>
           <div className='flex items-center'>
             <Link
               className={
                 'link flex-grow ellipsis ml3 pl1 ' +
-                  (caseItem.unreadCounts ? 'b ' : '') +
-                  (isClosed(caseItem) ? 'silver strike' : 'bondi-blue')
+                  (item.unreadCounts ? 'b ' : '') +
+                  (isClosed(item) ? 'silver strike' : 'bondi-blue')
               }
-              to={`/case/${caseItem.id}`}
+              to={name === 'report' ? (`report/${item.id}/review`) : (`/case/${item.id}`)}
```

Comment:
This component is pretty generic, so it could be used for other things, **aside** from this line. Please find a way to externalize all or something of the child's URL resolving to the parent components.